<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN">

<html>

<head>
<title>bibtex2html output</title>
</head>

<body>

<!-- This document was automatically generated with bibtex2html 1.99
     (see http://www.lri.fr/~filliatr/bibtex2html/),
     with the following command:
     bibtex2html -d -r -nodoi -nolinks  -->


<table>

<tr valign="top">
<td align="right" class="bibtexnumber">
[<a name="ramirez-gonzalez_transcriptional_2018">1</a>]
</td>
<td class="bibtexitem">
R.&nbsp;H. Ramirez-Gonzalez, P.&nbsp;Borrill, D.&nbsp;Lang, S.&nbsp;A. Harrington, J.&nbsp;Brinton,
  L.&nbsp;Venturini, M.&nbsp;Davey, J.&nbsp;Jacobs, F.&nbsp;van Ex, A.&nbsp;Pasha, Y.&nbsp;Khedikar, S.&nbsp;J.
  Robinson, A.&nbsp;T. Cory, T.&nbsp;Florio, L.&nbsp;Concia, C.&nbsp;Juery, H.&nbsp;Schoonbeek,
  B.&nbsp;Steuernagel, D.&nbsp;Xiang, C.&nbsp;J. Ridout, B.&nbsp;Chalhoub, K.&nbsp;F.&nbsp;X. Mayer,
  M.&nbsp;Benhamed, D.&nbsp;Latrasse, A.&nbsp;Bendahmane, International Wheat Genome
  Sequencing Consortium, B.&nbsp;B.&nbsp;H. Wulff, R.&nbsp;Appels, V.&nbsp;Tiwari, R.&nbsp;Datla,
  F.&nbsp;Choulet, C.&nbsp;J. Pozniak, N.&nbsp;J. Provart, A.&nbsp;G. Sharpe, E.&nbsp;Paux, M.&nbsp;Spannagl,
  A.&nbsp;Brautigam, and C.&nbsp;Uauy.
 The transcriptional landscape of polyploid wheat.
 <em>Science</em>, 361(6403):eaar6089, August 2018.

</td>
</tr>


<tr valign="top">
<td align="right" class="bibtexnumber">
[<a name="the_international_wheat_genome_sequencing_consortium_iwgsc_shifting_2018">2</a>]
</td>
<td class="bibtexitem">
The International Wheat Genome Sequencing Consortium (IWGSC), IWGSC RefSeq
  principal investigators:, Rudi Appels, Kellye Eversole, Catherine Feuillet,
  Beat Keller, Jane Rogers, Nils Stein, IWGSC whole-genome assembly principal
  investigators:, Curtis&nbsp;J. Pozniak, Nils Stein, Frederic Choulet, Assaf
  Distelfeld, Kellye Eversole, Jesse Poland, Jane Rogers, Gil Ronen, Andrew&nbsp;G.
  Sharpe, Whole-genome sequencing and assembly:, Curtis Pozniak, Gil Ronen,
  Nils Stein, Omer Barad, Kobi Baruch, Frederic Choulet, Gabriel
  Keeble-Gagnere, Martin Mascher, Andrew&nbsp;G. Sharpe, Gil Ben-Zvi, Ambre-Aurore
  Josselin, Hi-C data-based scaffolding:, Nils Stein, Martin Mascher, Axel
  Himmelbach, Whole-genome assembly quality control and analyses:, Frederic
  Choulet, Gabriel Keeble-Gagnere, Martin Mascher, Jane Rogers, Francois
  Balfourier, Juan Gutierrez-Gonzalez, Matthew Hayden, Ambre-Aurore Josselin,
  ChuShin Koh, Gary Muehlbauer, Raj&nbsp;K. Pasam, Etienne Paux, Curtis&nbsp;J. Pozniak,
  Philippe Rigault, Andrew&nbsp;G. Sharpe, Josquin Tibbits, Vijay Tiwari,
  Pseudomolecule assembly:, Frederic Choulet, Gabriel Keeble-Gagnere,
  Martin Mascher, Ambre-Aurore Josselin, Jane Rogers, RefSeq genome structure
  and gene analyses:, Manuel Spannagl, Frederic Choulet, Daniel Lang,
  Heidrun Gundlach, Georg Haberer, Gabriel Keeble-Gagnere, Klaus F.&nbsp;X. Mayer,
  Danara Ormanbekova, Etienne Paux, Verena Prade, Hana Simkova, Thomas
  Wicker, Automated annotation:, Frederic Choulet, Manuel Spannagl, David
  Swarbreck, Helene Rimbert, Marius Felder, Nicolas Guilhot, Heidrun
  Gundlach, Georg Haberer, Gemy Kaithakottil, Jens Keilwagen, Daniel Lang,
  Philippe Leroy, Thomas Lux, Klaus F.&nbsp;X. Mayer, Sven Twardziok, Luca
  Venturini, Manual gene curation:, Rudi Appels, Helene Rimbert, Frederic
  Choulet, Angela Juhasz, Gabriel Keeble-Gagnere, Subgenome comparative
  analyses:, Frederic Choulet, Manuel Spannagl, Daniel Lang, Michael Abrouk,
  Georg Haberer, Gabriel Keeble-Gagnere, Klaus F.&nbsp;X. Mayer, Thomas Wicker,
  Transposable elements:, Frederic Choulet, Thomas Wicker, Heidrun
  Gundlach, Daniel Lang, Manuel Spannagl, Phylogenomic analyses:, Daniel
  Lang, Manuel Spannagl, Rudi Appels, Iris Fischer, Transcriptome analyses and
  RNA-seq data:, Cristobal Uauy, Philippa Borrill, Ricardo&nbsp;H.
  Ramirez-Gonzalez, Rudi Appels, Dominique Arnaud, Smahane Chalabi, Boulos
  Chalhoub, Frederic Choulet, Aron Cory, Raju Datla, Mark&nbsp;W. Davey, Matthew
  Hayden, John Jacobs, Daniel Lang, Stephen&nbsp;J. Robinson, Manuel Spannagl,
  Burkhard Steuernagel, Josquin Tibbits, Vijay Tiwari, Fred van Ex, Brande
  B.&nbsp;H. Wulff, Whole-genome methylome:, Curtis&nbsp;J. Pozniak, Stephen&nbsp;J.
  Robinson, Andrew&nbsp;G. Sharpe, Aron Cory, Histone mark analyses:, Moussa
  Benhamed, Etienne Paux, Abdelhafid Bendahmane, Lorenzo Concia, David
  Latrasse, BAC chromosome MTP IWGSC-Bayer Whole-Genome Profiling (WGP)
  tags:, Jane Rogers, John Jacobs, Michael Alaux, Rudi Appels, Jan Bartos,
  Arnaud Bellec, Helene Berges, Jaroslav Dolezel, Catherine Feuillet, Zeev
  Frenkel, Bikram Gill, Abraham Korol, Thomas Letellier, Odd-Arne Olsen, Hana
  Simkova, Kuldeep Singh, Miroslav Valarik, Edwin van&nbsp;der Vossen, Sonia
  Vautrin, Song Weining, Chromosome LTC mapping and physical mapping quality
  control:, Abraham Korol, Zeev Frenkel, Tzion Fahima, Vladimir Glikson, Dina
  Raats, Jane Rogers, RH mapping:, Vijay Tiwari, Bikram Gill, Etienne Paux,
  Jesse Poland, Optical mapping:, Jaroslav Dolezel, Jarmila Cihalikova,
  Hana Simkova, Helena Toegelova, Jan Vrana, Recombination analyses:,
  Pierre Sourdille, Benoit Darrier, Gene family analyses:, Rudi Appels,
  Manuel Spannagl, Daniel Lang, Iris Fischer, Danara Ormanbekova, Verena Prade,
  CBF gene family:, Delfina Barabaschi, Luigi Cattivelli, Dehydrin gene
  family:, Pilar Hernandez, Sergio Galvez, Hikmet Budak, NLR gene family:,
  Burkhard Steuernagel, Jonathan D.&nbsp;G. Jones, Kamil Witek, Brande B.&nbsp;H. Wulff,
  Guotai Yu, PPR gene family:, Ian Small, Joanna Melonek, Ruonan Zhou,
  Prolamin gene family:, Angela Juhasz, Tatiana Belova, Rudi Appels,
  Odd-Arne Olsen, WAK gene family:, Kostya Kanyuka, Robert King, Stem
  solidness (SSt1) QTL team:, Kirby Nilsen, Sean Walkowiak, Curtis&nbsp;J. Pozniak,
  Richard Cuthbert, Raju Datla, Ron Knox, Krysta Wiebe, Daoquan Xiang,
  Flowering locus C (FLC) gene team:, Antje Rohde, Timothy Golds, Genome
  size analysis:, Jaroslav Dolezel, Jana Cizkova, Josquin Tibbits,
  MicroRNA and tRNA annotation:, Hikmet Budak, Bala&nbsp;Ani Akpinar, Sezgi
  Biyiklioglu, Genetic maps and mapping:, Gary Muehlbauer, Jesse Poland,
  Liangliang Gao, Juan Gutierrez-Gonzalez, Amidou N'Daiye, BAC libraries and
  chromosome sorting:, Jaroslav Dolezel, Hana Simkova, Jarmila
  Cihalikova, Marie Kubalakova, Jan Safar, Jan Vrana, BAC pooling,
  BAC library repository, and access:, Helene Berges, Arnaud Bellec, Sonia
  Vautrin, IWGSC sequence and data repository and access:, Michael Alaux,
  Francoise Alfama, Anne-Francoise Adam-Blondon, Raphael Flores, Claire
  Guerche, Thomas Letellier, Mikael Loaec, Hadi Quesneville, Physical maps
  and BAC-based sequences:, 1A BAC sequencing and assembly:, Curtis&nbsp;J.
  Pozniak, Andrew&nbsp;G. Sharpe, Sean Walkowiak, Hikmet Budak, Janet Condie,
  Jennifer Ens, ChuShin Koh, Ron Maclachlan, Yifang Tan, Thomas Wicker, 1B BAC
  sequencing and assembly:, Frederic Choulet, Etienne Paux, Adriana Alberti,
  Jean-Marc Aury, Francois Balfourier, Valerie Barbe, Arnaud Couloux, Corinne
  Cruaud, Karine Labadie, Sophie Mangenot, Patrick Wincker, 1D, 4D, and 6D
  physical mapping:, Bikram Gill, Gaganpreet Kaur, Mingcheng Luo, Sunish
  Sehgal, 2AL physical mapping:, Kuldeep Singh, Parveen Chhuneja, Om&nbsp;Prakash
  Gupta, Suruchi Jindal, Parampreet Kaur, Palvi Malik, Priti Sharma, Bharat
  Yadav, 2AS physical mapping:, Nagendra&nbsp;K. Singh, JitendraP. Khurana,
  Chanderkant Chaudhary, Paramjit Khurana, Vinod Kumar, Ajay Mahato, Saloni
  Mathur, Amitha Sevanthi, Naveen Sharma, Ram&nbsp;Sewak Tomar, 2B, 2D, 4B, 5BL,
  and 5DL IWGSC-Bayer Whole-Genome Profiling (WGP) physical maps:, Jane
  Rogers, John Jacobs, Michael Alaux, Arnaud Bellec, Helene Berges, Jaroslav
  Dolezel, Catherine Feuillet, Zeev Frenkel, Bikram Gill, Abraham Korol, Edwin
  van&nbsp;der Vossen, Sonia Vautrin, 3AL physical mapping:, Bikram Gill,
  Gaganpreet Kaur, Mingcheng Luo, Sunish Sehgal, 3DS physical mapping and BAC
  sequencing and assembly:, Jan Bartos, Katerina Holusova, Ondrej
  Plihal, 3DL BAC sequencing and assembly:, Matthew&nbsp;D. Clark, Darren
  Heavens, George Kettleborough, Jon Wright, 4A physical mapping, BAC
  sequencing, assembly, and annotation:, Miroslav Valarik, Michael Abrouk,
  Barbora Balcarkova, Katerina Holusova, Yuqin Hu, Mingcheng Luo, 5BS BAC
  sequencing and assembly:, Elena Salina, Nikolai Ravin, Konstantin Skryabin,
  Alexey Beletsky, Vitaly Kadnikov, Andrey Mardanov, Michail Nesterov, Andrey
  Rakitin, Ekaterina Sergeeva, 6B BAC sequencing and assembly:, Hirokazu
  Handa, Hiroyuki Kanamori, Satoshi Katagiri, Fuminori Kobayashi, Shuhei
  Nasuda, Tsuyoshi Tanaka, Jianzhong Wu, 7A physical mapping and BAC
  sequencing:, Rudi Appels, Matthew Hayden, Gabriel Keeble-Gagnere, Philippe
  Rigault, Josquin Tibbits, 7B physical mapping, BAC sequencing, and
  assembly:, Odd-Arne Olsen, Tatiana Belova, Federica Cattonaro, Min Jiumeng,
  Karl Kugler, Klaus F.&nbsp;X. Mayer, Matthias Pfeifer, Simen Sandve, Xu&nbsp;Xun, Bujie
  Zhan, 7DS BAC sequencing and assembly:, Hana Simkova, Michael Abrouk,
  Jacqueline Batley, Philipp&nbsp;E. Bayer, David Edwards, Satomi Hayashi, Helena
  Toegelova, Zuzana Tulpova, Paul Visendi, 7DL physical mapping and BAC
  sequencing:, Song Weining, Licao Cui, Xianghong Du, Kewei Feng, Xiaojun Nie,
  Wei Tong, Le&nbsp;Wang, Figures:, Philippa Borrill, Heidrun Gundlach, Sergio
  Galvez, Gemy Kaithakottil, Daniel Lang, Thomas Lux, Martin Mascher, Danara
  Ormanbekova, Verena Prade, Ricardo&nbsp;H. Ramirez-Gonzalez, Manuel Spannagl, Nils
  Stein, Cristobal Uauy, Luca Venturini, Manuscript writing team:, Nils
  Stein, Rudi Appels, Kellye Eversole, Jane Rogers, Philippa Borrill, Luigi
  Cattivelli, Frederic Choulet, Pilar Hernandez, Kostya Kanyuka, Daniel Lang,
  Martin Mascher, Kirby Nilsen, Etienne Paux, Curtis&nbsp;J. Pozniak, Ricardo&nbsp;H.
  Ramirez-Gonzalez, Hana Simkova, Ian Small, Manuel Spannagl, David
  Swarbreck, and Cristobal Uauy.
 Shifting the limits in wheat research and breeding using a fully
  annotated reference genome.
 <em>Science</em>, 361(6403):eaar7191, August 2018.

</td>
</tr>


<tr valign="top">
<td align="right" class="bibtexnumber">
[<a name="melonek2018high">3</a>]
</td>
<td class="bibtexitem">
Joanna Melonek, Ruonan Zhou, Philipp&nbsp;E Bayer, David Edwards, Nils Stein, and
  Ian Small.
 High intraspecific diversity of restorer-of-fertility-like genes in
  barley.
 <em>The Plant Journal</em>, 2018.

</td>
</tr>


<tr valign="top">
<td align="right" class="bibtexnumber">
[<a name="bayer2018bias">4</a>]
</td>
<td class="bibtexitem">
Philipp&nbsp;E Bayer, David Edwards, and Jacqueline Batley.
 Bias in resistance gene prediction due to repeat masking.
 <em>Nature Plants</em>, 4(10):762, 2018.

</td>
</tr>


<tr valign="top">
<td align="right" class="bibtexnumber">
[<a name="bayer2018variation">5</a>]
</td>
<td class="bibtexitem">
Philipp&nbsp;E Bayer, Agnieszka&nbsp;A Golicz, Soodeh Tirnaz, Chon-Kit&nbsp;Kenneth Chan,
  David Edwards, and Jacqueline Batley.
 Variation in abundance of predicted resistance genes in the brassica
  oleracea pangenome.
 <em>Plant biotechnology journal</em>, 2018.

</td>
</tr>


<tr valign="top">
<td align="right" class="bibtexnumber">
[<a name="mousavi2018western">6</a>]
</td>
<td class="bibtexitem">
Mahsa Mousavi-Derazmahalleh, Bruno Nevado, Philipp&nbsp;E Bayer, Dmitry&nbsp;A Filatov,
  James&nbsp;K Hane, David Edwards, William Erskine, and Matthew&nbsp;N Nelson.
 The western mediterranean region provided the founder population of
  domesticated narrow-leafed lupin.
 <em>Theoretical and Applied Genetics</em>, pages 1--12, 2018.

</td>
</tr>


<tr valign="top">
<td align="right" class="bibtexnumber">
[<a name="mousavi_derazmahalleh_adapting_2018">7</a>]
</td>
<td class="bibtexitem">
Mahsa Mousavi-Derazmahalleh, Philipp&nbsp;E. Bayer, James&nbsp;K. Hane, Valliyodan Babu,
  Henry&nbsp;T. Nguyen, Matthew&nbsp;N. Nelson, William Erskine, Rajeev&nbsp;K. Varshney,
  Roberto Papa, and David Edwards.
 Adapting legume crops to climate change using genomic approaches:
  climate change legumes.
 <em>Plant, Cell &amp; Environment</em>, 2018.
 Impact Factor:6.173.

</td>
</tr>


<tr valign="top">
<td align="right" class="bibtexnumber">
[<a name="lee_genomic_2018">8</a>]
</td>
<td class="bibtexitem">
HueyTyng Lee, Agnieszka&nbsp;A Golicz, Philipp&nbsp;E Bayer, Anita Severn-Ellis, Chon-Kit
  Kenneth&nbsp;Chan, Jacqueline Batley, Gary&nbsp;A Kendrick, and David Edwards.
 Genomic Comparison of Two Independent Seagrass Lineages
  Reveals Habitat-driven Convergent Evolution.
 <em>Journal of Experimental Botany</em>, 2018.
 Impact factor: 5.830.

</td>
</tr>


<tr valign="top">
<td align="right" class="bibtexnumber">
[<a name="taylor_indel_2018">9</a>]
</td>
<td class="bibtexitem">
Candy&nbsp;M. Taylor, Lars&nbsp;G. Kamphuis, Weilu Zhang, Gagan Garg, Jens&nbsp;D. Berger,
  Mahsa Mousavi-Derazmahalleh, Philipp&nbsp;E. Bayer, David Edwards, Karam&nbsp;B. Singh,
  Wallace&nbsp;A. Cowling, and Matthew&nbsp;N. Nelson.
 INDEL variation in the regulatory region of the major flowering
  time gene <i>LanFTc1</i> is associated with vernalization response and
  flowering time in narrow-leafed lupin ( <i>Lupi nus angustifolius</i>
  L.): INDEL variation in the regulatory region of <i>LanFTc1</i>.
 <em>Plant, Cell &amp; Environment</em>, 2018.
 Impact Factor: 6.173.

</td>
</tr>


<tr valign="top">
<td align="right" class="bibtexnumber">
[<a name="mousavi2018exploring">10</a>]
</td>
<td class="bibtexitem">
Mahsa Mousavi-Derazmahalleh, Philipp&nbsp;E Bayer, Bruno Nevado, Bhavna Hurgobin,
  Dmitry Filatov, Andrzej Kilian, Lars&nbsp;G Kamphuis, Karam&nbsp;B Singh, Jens&nbsp;D
  Berger, James&nbsp;K Hane, et&nbsp;al.
 Exploring the genetic and adaptive diversity of a pan-mediterranean
  crop wild relative: narrow-leafed lupin.
 <em>Theoretical and Applied Genetics</em>, 131(4):887--901, 2018.

</td>
</tr>


<tr valign="top">
<td align="right" class="bibtexnumber">
[<a name="hurgobin2018homoeologous">11</a>]
</td>
<td class="bibtexitem">
Bhavna Hurgobin, Agnieszka&nbsp;A Golicz, Philipp&nbsp;E Bayer, Chon-Kit&nbsp;Kenneth Chan,
  Soodeh Tirnaz, Aria Dolatabadian, Sarah&nbsp;V Schiessl, Birgit Samans, Juan&nbsp;D
  Montenegro, Isobel&nbsp;AP Parkin, et&nbsp;al.
 Homoeologous exchange is a major cause of gene presence/absence
  variation in the amphidiploid brassica napus.
 <em>Plant Biotechnology Journal</em>, 16(7):1265--1274, 2018.
 Impact Factor: 6.09, JCR Plant Sciences 11/209 (Q1), Altmetric score:
  6.

</td>
</tr>


<tr valign="top">
<td align="right" class="bibtexnumber">
[<a name="kaur2017climate">12</a>]
</td>
<td class="bibtexitem">
Parwinder Kaur, Rudi Appels, Philipp&nbsp;E Bayer, Gabriel Keeble-Gagnere, Jiankang
  Wang, Hideki Hirakawa, Kenta Shirasawa, Philip Vercoe, Katia Stefanova, Zoey
  Durmic, et&nbsp;al.
 Climate clever clovers: New paradigm to reduce the environmental
  footprint of ruminants by breeding low methanogenic forages utilizing
  haplotype variation.
 <em>Frontiers in Plant Science</em>, 8:1463, 2017.

</td>
</tr>


<tr valign="top">
<td align="right" class="bibtexnumber">
[<a name="yuan2017bionanoanalyst">13</a>]
</td>
<td class="bibtexitem">
Yuxuan Yuan, Philipp&nbsp;E Bayer, Armin Scheben, Chon-Kit&nbsp;Kenneth Chan, and David
  Edwards.
 Bionanoanalyst: a visualisation tool to assess genome assembly
  quality using bionano data.
 <em>BMC Bioinformatics</em>, 18(1):323, 2017.
 Impact factor: 2.4.

</td>
</tr>


<tr valign="top">
<td align="right" class="bibtexnumber">
[<a name="yuan2017runbng">14</a>]
</td>
<td class="bibtexitem">
Yuxuan Yuan, Philipp&nbsp;E Bayer, Huey-Tyng Lee, and David Edwards.
 runbng: a software package for bionano genomic analysis on the
  command line.
 <em>Bioinformatics</em>, 33(19):3107--3109, 2017.
 Impact factor: 7.3.

</td>
</tr>


<tr valign="top">
<td align="right" class="bibtexnumber">
[<a name="bayer2017assembly">15</a>]
</td>
<td class="bibtexitem">
Philipp&nbsp;E Bayer, Bhavna Hurgobin, Agnieszka&nbsp;A Golicz, Chon-Kit&nbsp;Kenneth Chan,
  Yuxuan Yuan, HueyTyng Lee, Michael Renton, Jinling Meng, Ruiyuan Li, Yan
  Long, et&nbsp;al.
 Assembly and comparison of two closely related <i>Brassica
  napus</i> genomes.
 <em>Plant Biotechnology Journal</em>, 15(12):1602--1610, 2017.
 Impact Factor: 6.09, JCR Plant Sciences 11/209 (Q1), Altmetric score:
  11.

</td>
</tr>


<tr valign="top">
<td align="right" class="bibtexnumber">
[<a name="yuan2017improvements">16</a>]
</td>
<td class="bibtexitem">
Yuxuan Yuan, Philipp&nbsp;E. Bayer, Jacqueline Batley, and David Edwards.
 Improvements in genomic technologies: Application to crop genomics.
 <em>Trends in Biotechnology</em>, 35(6):547--558, 2017.
 Impact Factor: 12.06, JCR Biotechnology and Applied Microbiology
  3/161 (Q1), Altmetric score: 7.

</td>
</tr>


<tr valign="top">
<td align="right" class="bibtexnumber">
[<a name="montenegro2017pangenome">17</a>]
</td>
<td class="bibtexitem">
Juan&nbsp;D. Montenegro, Agnieszka&nbsp;A. Golicz, Philipp&nbsp;E. Bayer, Bhavna Hurgobin,
  HueyTyng Lee, Chon-Kit&nbsp;Kenneth Chan, Paul Visendi, Kaitao Lai, Jaroslav
  Dolezel, Jacqueline Batley, and David Edwards.
 The pangenome of hexaploid bread wheat.
 <em>The Plant Journal</em>, 90(5):1007--1013, 2017.
 Best paper award (Plant Journal) 2017 Resource - Bronze, Impact
  Factor: 5.46, JCR Plant Sciences 13/209 (Q1), Altmetric score: 52.

</td>
</tr>


<tr valign="top">
<td align="right" class="bibtexnumber">
[<a name="kaur2017advanced">18</a>]
</td>
<td class="bibtexitem">
Parwinder Kaur, Philipp&nbsp;E. Bayer, Zbynek Milec, Jan Vrana, Yuxuan Yuan, Rudi
  Appels, David Edwards, Jacqueline Batley, Phillip Nichols, William Erskine,
  and Jaroslav Dolezel.
 An advanced reference genome of <i>Trifolium subterraneum</i> l.
  reveals genes related to agronomic performance.
 <em>Plant Biotechnology Journal</em>, 15(8):1034--1046, 2017.
 Impact Factor: 6.09, JCR Plant Sciences 11/209 (Q1), Altmetric score:
  15.

</td>
</tr>


<tr valign="top">
<td align="right" class="bibtexnumber">
[<a name="hane2017comprehensive">19</a>]
</td>
<td class="bibtexitem">
James&nbsp;K. Hane, Yao Ming, Lars&nbsp;G. Kamphuis, Matthew&nbsp;N. Nelson, Gagan Garg,
  Craig&nbsp;A. Atkins, Philipp&nbsp;E. Bayer, Armando Bravo, Scott Bringans, Steven
  Cannon, David Edwards, Rhonda Foley, Ling-ling Gao, Maria&nbsp;J. Harrison, Wei
  Huang, Bhavna Hurgobin, Sean Li, Cheng-Wu Liu, Annette McGrath, Grant
  Morahan, Jeremy Murray, James Weller, Jianbo Jian, and Karam&nbsp;B. Singh.
 A comprehensive draft genome sequence for lupin (<i>Lupinus
  angustifolius</i>), an emerging health food: insights into plant--microbe
  interactions and legume evolution.
 <em>Plant Biotechnology Journal</em>, 15(3):318--330, 2017.
 Citations: 6, Impact Factor: 6.09, JCR Plant Sciences 11/209 (Q1),
  Altmetric score: 15.

</td>
</tr>


<tr valign="top">
<td align="right" class="bibtexnumber">
[<a name="gacek2016genome">20</a>]
</td>
<td class="bibtexitem">
Katarzyna Gacek, Philipp&nbsp;E Bayer, Iwona Bartkowiak-Broda, Laurencja Szala,
  Jaroslav Bocianowski, David Edwards, and Jacqueline Batley.
 Genome-wide association study of genetic control of seed fatty acid
  biosynthesis in <i>Brassica napus</i>.
 <em>Frontiers in Plant Science</em>, 7:2062, 2016.
 Impact Factor: 4.49, JCR Plant Sciences 15/209 (Q1).

</td>
</tr>


<tr valign="top">
<td align="right" class="bibtexnumber">
[<a name="golicz2016pangenome">21</a>]
</td>
<td class="bibtexitem">
Agnieszka&nbsp;A. Golicz, Philipp&nbsp;E. Bayer, Guy&nbsp;C. Barker, Patrick&nbsp;P. Edger, HyeRan
  Kim, Paula&nbsp;A. Martinez, Chon Kit&nbsp;Kenneth Chan, Anita Severn-Ellis, W.&nbsp;Richard
  McCombie, Isobel A.&nbsp;P. Parkin, Andrew&nbsp;H. Paterson, J.&nbsp;Chris Pires, Andrew&nbsp;G.
  Sharpe, Haibao Tang, Graham&nbsp;R. Teakle, Christopher&nbsp;D. Town, Jacqueline
  Batley, and David Edwards.
 The pangenome of an agronomically important crop plant
  <i>Brassica oleracea</i>.
 <em>Nature Communications</em>, 7:13390, 2016.
 Citations: 5, Impact Factor: 11.32, JCR Multidisciplinary Sciences
  3/63 (Q1), Altmetric score: 50.

</td>
</tr>


<tr valign="top">
<td align="right" class="bibtexnumber">
[<a name="barash2016candidate">22</a>]
</td>
<td class="bibtexitem">
Mark Barash, Philipp&nbsp;E Bayer, and Angela van Daal.
 Candidate gene scan for single nucleotide polymorphisms involved in
  the determination of normal variability in human craniofacial morphology.
 <em>bioRxiv</em>, page 060814, 2016.
 Altmetric score: 9.

</td>
</tr>


<tr valign="top">
<td align="right" class="bibtexnumber">
[<a name="lee2016genome">23</a>]
</td>
<td class="bibtexitem">
HueyTyng Lee, Agnieszka&nbsp;A. Golicz, Philipp Bayer, Yuannian Jiao, Haibao Tang,
  Andrew&nbsp;H. Paterson, Gaurav Sablok, Rahul&nbsp;R. Krishnaraj, Chon-Kit&nbsp;Kenneth
  Chan, Jacqueline Batley, Gary&nbsp;A. Kendrick, Anthony&nbsp;William Larkum, Peter&nbsp;J
  Ralph, and David Edwards.
 The genome of a southern hemisphere seagrass species <i>Zostera
  muelleri</i>.
 <em>Plant Physiology</em>, page 868, 2016.
 Citations; 3, Impact Factor: 6.28, JCR Plant Sciences 8/209 (Q1),
  Altmetric score: 6.

</td>
</tr>


<tr valign="top">
<td align="right" class="bibtexnumber">
[<a name="bayer2016genomics">24</a>]
</td>
<td class="bibtexitem">
Philipp&nbsp;E Bayer.
 Genomics of salinity.
 In David Edwards and Jacqueline Batley, editors, <em>Plant Genomics
  and Climate Change</em>, pages 179--1940.3. Springer New
  York, 2016.

</td>
</tr>


<tr valign="top">
<td align="right" class="bibtexnumber">
[<a name="visendi2016efficient">25</a>]
</td>
<td class="bibtexitem">
Paul Visendi, Paul&nbsp;J Berkman, Satomi Hayashi, Agnieszka&nbsp;A Golicz, Philipp&nbsp;E
  Bayer, Pradeep Ruperao, Bhavna Hurgobin, Juan Montenegro, Chon-Kit&nbsp;Kenneth
  Chan, Helena Sta&#328;kov&aacute;, et&nbsp;al.
 An efficient approach to bac based assembly of complex genomes.
 <em>Plant methods</em>, 12(1):2, 2016.
 Impact Factor: 3.4, JCR Plant Sciences 30/209 (Q1).

</td>
</tr>


<tr valign="top">
<td align="right" class="bibtexnumber">
[<a name="mason2016centromere">26</a>]
</td>
<td class="bibtexitem">
A.&nbsp;S. Mason, M.&nbsp;Rousseau-Gueutin, J.&nbsp;Morice, P.&nbsp;E. Bayer, N.&nbsp;Besharat,
  A.&nbsp;Cousin, A.&nbsp;Pradhan, I.&nbsp;A.&nbsp;P. Parkin, A.-M. Chevre, J.&nbsp;Batley, and M.&nbsp;N.
  Nelson.
 Centromere locations in <i>Brassica</i> a and c genomes revealed
  through half-tetrad analysis.
 <em>Genetics</em>, 202(2):513--523, 2016.
 Citations: 8, Impact Factor: 4.6, JCR Genetics &amp; Heredity 29/166
  (Q1).

</td>
</tr>


<tr valign="top">
<td align="right" class="bibtexnumber">
[<a name="bayer2016skim">27</a>]
</td>
<td class="bibtexitem">
Philipp&nbsp;E Bayer.
 Skim-based genotyping by sequencing using a double haploid population
  to call snps, infer gene conversions, and improve genome assemblies.
 In David Edwards, editor, <em>Plant Bioinformatics: Methods and
  Protocols</em>, pages 285--292. Springer New York, 2016.

</td>
</tr>


<tr valign="top">
<td align="right" class="bibtexnumber">
[<a name="bayer2015high">28</a>]
</td>
<td class="bibtexitem">
Philipp&nbsp;E Bayer, Pradeep Ruperao, Annaliese&nbsp;S Mason, Jiri Stiller,
  Chon-Kit&nbsp;Kenneth Chan, Satomi Hayashi, Yan Long, Jinling Meng, Tim Sutton,
  Paul Visendi, et&nbsp;al.
 High-resolution skim genotyping by sequencing reveals the
  distribution of crossovers and gene conversions in <i>Cicer arietinum</i>
  and <i>Brassica napus</i>.
 <em>Theoretical and Applied Genetics</em>, 128(6):1039--1047, 2015.
 Citations: 14, Impact Factor: 3.9, JCR Agronomy 5/83 (Q1).

</td>
</tr>


<tr valign="top">
<td align="right" class="bibtexnumber">
[<a name="golicz2015skim">29</a>]
</td>
<td class="bibtexitem">
Agnieszka&nbsp;A Golicz, Philipp&nbsp;E Bayer, and David Edwards.
 Skim-based genotyping by sequencing.
 In David Edwards, editor, <em>Plant Genotyping: Methods and
  Protocols</em>, pages 257--270. Springer New York, 2015.
 Citations: 6.

</td>
</tr>


<tr valign="top">
<td align="right" class="bibtexnumber">
[<a name="lai2015identification">30</a>]
</td>
<td class="bibtexitem">
Kaitao Lai, Michal&nbsp;T Lorenc, Hong&nbsp;Ching Lee, Paul&nbsp;J Berkman, Philipp&nbsp;E
  Bayer, Paul Visendi, Pradeep Ruperao, Timothy&nbsp;L Fitzgerald, Manuel Zander,
  Chon-Kit&nbsp;Kenneth Chan, et&nbsp;al.
 Identification and characterization of more than 4 million
  intervarietal snps across the group 7 chromosomes of bread wheat.
 <em>Plant Biotechnology Journal</em>, 13(1):97--104, 2015.
 Citations: 17, Impact Factor: 6.09, JCR Plant Sciences 11/209 (Q1).

</td>
</tr>


<tr valign="top">
<td align="right" class="bibtexnumber">
[<a name="chalhoub2014early">31</a>]
</td>
<td class="bibtexitem">
B.&nbsp;Chalhoub, F.&nbsp;Denoeud, S.&nbsp;Liu, I.&nbsp;A.&nbsp;P. Parkin, H.&nbsp;Tang, X.&nbsp;Wang, J.&nbsp;Chiquet,
  H.&nbsp;Belcram, C.&nbsp;Tong, B.&nbsp;Samans, M.&nbsp;Correa, C.&nbsp;Da Silva, J.&nbsp;Just,
  C.&nbsp;Falentin, C.&nbsp;S. Koh, I.&nbsp;Le Clainche, M.&nbsp;Bernard, P.&nbsp;Bento, B.&nbsp;Noel,
  K.&nbsp;Labadie, A.&nbsp;Alberti, M.&nbsp;Charles, D.&nbsp;Arnaud, H.&nbsp;Guo, C.&nbsp;Daviaud,
  S.&nbsp;Alamery, K.&nbsp;Jabbari, M.&nbsp;Zhao, P.&nbsp;P. Edger, H.&nbsp;Chelaifa, D.&nbsp;Tack,
  G.&nbsp;Lassalle, I.&nbsp;Mestiri, N.&nbsp;Schnel, M.-C. Le Paslier, G.&nbsp;Fan, V.&nbsp;Renault,
  P.&nbsp;E. Bayer, A.&nbsp;A. Golicz, S.&nbsp;Manoli, T.-H. Lee, V.&nbsp;H.&nbsp;D. Thi, S.&nbsp;Chalabi,
  Q.&nbsp;Hu, C.&nbsp;Fan, R.&nbsp;Tollenaere, Y.&nbsp;Lu, C.&nbsp;Battail, J.&nbsp;Shen, C.&nbsp;H.&nbsp;D.
  Sidebottom, A.&nbsp;Canaguier, A.&nbsp;Chauveau, A.&nbsp;Berard, G.&nbsp;Deniot, M.&nbsp;Guan, Z.&nbsp;Liu,
  F.&nbsp;Sun, Y.&nbsp;P. Lim, E.&nbsp;Lyons, C.&nbsp;D. Town, I.&nbsp;Bancroft, J.&nbsp;Meng, J.&nbsp;Ma, J.&nbsp;C.
  Pires, G.&nbsp;J. King, D.&nbsp;Brunel, R.&nbsp;Delourme, M.&nbsp;Renard, J.-M. Aury, K.&nbsp;L.
  Adams, J.&nbsp;Batley, R.&nbsp;J. Snowdon, J.&nbsp;Tost, D.&nbsp;Edwards, Y.&nbsp;Zhou, W.&nbsp;Hua, A.&nbsp;G.
  Sharpe, A.&nbsp;H. Paterson, C.&nbsp;Guan, and P.&nbsp;Wincker.
 Early allopolyploid evolution in the post-neolithic <i>Brassica
  napus</i> oilseed genome.
 <em>Science</em>, 345(6199):950--953, 2014.
 Citations: 384, Impact Factor: 34.6, JCR Multidisciplinary Sciences
  2/62 (Q1), Altmetric score: 175.

</td>
</tr>


<tr valign="top">
<td align="right" class="bibtexnumber">
[<a name="mason2014high">32</a>]
</td>
<td class="bibtexitem">
Annaliese&nbsp;S Mason, Jacqueline Batley, Philipp&nbsp;E Bayer, Alice Hayward, Wallace&nbsp;A
  Cowling, and Matthew&nbsp;N Nelson.
 High-resolution molecular karyotyping uncovers pairing between
  ancestrally related <i>Brassica</i> chromosomes.
 <em>New Phytologist</em>, 202(3):964--974, 2014.
 Citations: 16, Impact Factor: 7.21, JCR Plant Sciences 5/209 (Q1).

</td>
</tr>


<tr valign="top">
<td align="right" class="bibtexnumber">
[<a name="greshake2014opensnp">33</a>]
</td>
<td class="bibtexitem">
Bastian Greshake, Philipp&nbsp;E Bayer, Helge Rausch, and Julia Reda.
 Opensnp--a crowdsourced web resource for personal genomics.
 <em>PLoS One</em>, 9(3):e89204, 2014.
 Citations: 26, Impact Factor: 3.057, JCR Multidisciplinary Sciences
  11/63 (Q1), Altmetric score: 61.

</td>
</tr>


<tr valign="top">
<td align="right" class="bibtexnumber">
[<a name="dattolo2013acclimation">34</a>]
</td>
<td class="bibtexitem">
Emanuela Dattolo, Jenny Gu, Philipp&nbsp;E Bayer, Silvia Mazzuca, Ilia&nbsp;Anna Serra,
  Antonia Spadafora, Letizia Bernardo, Lucia Natali, Andrea Cavallini, and
  Gabriele Procaccini.
 Acclimation to different depths by the marine angiosperm
  <i>Posidonia oceanica</i>: transcriptomic and proteomic profiles.
 <em>Frontiers in Plant Science</em>, 4:195, 2013.

</td>
</tr>
</table><hr><p><em>This file was generated by
<a href="http://www.lri.fr/~filliatr/bibtex2html/">bibtex2html</a> 1.99.</em></p>
</body>
</html>
